tests/Feature/UserAuthenticationTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Support\Facades\Auth;
+use Tests\TestCase;
+
+class UserAuthenticationTest extends TestCase
+{
+    private const URL_STATUS = '/api/v1/user/status';
+    private const URL_LOGIN = '/api/v1/user/login';
+    private const URL_LOGOUT = '/api/v1/user/logout';
+
+    use DatabaseTransactions;
+
+    public function testStatusWithUnauthorizedUserWillFail(): void
+    {
+        $result = $this->get(static::URL_STATUS);
+
+        $this->assertEquals('exception.unauthorised', $result->json('errors')[0]['code']);
+        $result->assertUnauthorized();
+        $result->assertJsonStructure($this->unauthorizedErrorJsonStructure());
+    }
+
+    public function testStatusWithAuthorizedUserWillPass(): void
+    {
+        /** @var User $user */
+        $user = User::factory()->create();
+        Auth::setUser($user);
+
+        $result = $this->get(static::URL_STATUS);
+
+        $this->assertEquals($user->id, $result->json('data.user.id'));
+        $result->assertOk();
+        $result->assertJsonStructure($this->loggedUserStatusJsonStructure());
+    }
+
+    public function testLoginWithInvalidDataWillFail(): void
```

Comment:
tutaj wyskoczył mi błąd: 
`file_get_contents(../resources/lang/pl_PL/validation.json): Failed to open stream: No such file or directory`
i nie jestem pewien czy to kwestia czegoś u mnie (za pośrednictwem strzału bezpośrednio przez API tego błędu nie miałem)

Reply:
może być, że dostawanie się do resources po relatywnej ścieżce jest w sumie błędne, zbadam